js/items.js

```diff
@@ -0,0 +1,123 @@
+class ItemEffect {
+    constructor(HP, strength, endurance, DMGbonus, DMSresist, target) {
+        this.target = target;
+        this.HP = HP;
+        this.strength = strength;
+        this.endurance = endurance;
+        this.DMGbonus = DMGbonus;
+        this.DMSresist = DMSresist;
+    }
+}
+
+export class Equipment extends ItemEffect{
+    constructor(name, spriteIMG, type, HP, strength, endurance, DMGbonus, DMSresist, price) {
+        super(HP, strength, endurance, DMGbonus, DMSresist, "player");
+        
+        this.name = name;
+        this.spriteIMG = spriteIMG;
+        this.type = type;
+        this.price = price;
+        
+        const temp = this.generateEquipmentDescription();
+        this.firstDescription = temp.firstDescription;
+        this.secondDescription = temp.secondDescription;
+        this.priceDescription = temp.priceDescription;
+    }
+
+    generateEquipmentDescription() {
+        let inner = [];
+        let type ;
+        if (this.DMGbonus !== null) {
+            inner[1] = `Increases player DMG by ${this.DMGbonus * 100}%`;
```

Comment:
Let's use "damage" word
```suggestion
            inner[1] = `Increases player damage by ${this.DMGbonus * 100}%`;
```